components/popover/popover.jsx

```diff
@@ -570,7 +549,7 @@ const Popover = createReactClass({
 
 		this.renderOverlay(this.getIsOpen());
 
-		const containerStyles = { display: 'inline' };
+		const containerStyles = { display: 'inline-block' }; // ATTN FIXME - can we make this change??!
```

Comment:
I think it will be OK. `slds-button` should already be inline block

```
/**
 * @summary This neutralizes all the base styles making it look like a text link
 *
 * @name base
 * @selector .slds-button
 * @restrict button, a, span
 * @variant
 */
.slds-button {
  position: relative;
  display: inline-block;
  padding: 0;
```

Reply:
Oh i need to remove the comment